docs/validators/index.md

```diff
@@ -33,6 +33,25 @@ To have a **validator permit** in a given subnet, allowing you to submit miner e
 
 - You must be one of the top 64 validators in the subnet, ranked by stake weight.
 
+## Hotkey Association & Staking (subnet 0, the root subnet, only)
+
+:::tip Root Subnet (Subnet 0) only
+Skip this step if you are not registering a validator on the root subnet (subnet 0)
+:::
+
+```bash
+btcli wallet associate-hotkey --wallet.name  <wallet name> --hotkey <your hotkey>
+```
+Add stake to your validator before registering: 
+
+```bash
+# Stake funds to your hotkey account within the subnet.
+btcli stake add --wallet.name <wallet name> --wallet.hotkey <your validating hotkey>
+```
+:::tip Root Subnet (Subnet 0) only
```

Comment:
don't think this is necessary, as the stake weight requirement is covered elsewhere on the page.
 also, it doesn't really make sense here as you say 'subnet 0 only', so there is no alpha, and there is no weight-setting on subnet 0.